bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -78,6 +76,21 @@ limitations under the License.
       <version>${hbase.version}</version>
     </dependency>
 
+    <!-- For HBase 2.x, this should be hbase-mapreduce
```

Comment:
Thanks @igorbernstein2 ,

Could you elaborate on how can I find out about this my own? "hbase-shaded-server includes hbase-shaded-client"

I checked `hbase-shaded-server` in maven repo page and didn't see a indirect dependency.
Also didn't find anything from the output of effective-pom.

Is it possible that the hbase-shaded-client is brought in by the two dependencies you mentioned?

Reply:
hbase-shaded-server is the shaded version of hbase-server. hbase-server has a direct dependency on hbase-client. Since hbase-shaded-server doesnt declare it, then you can guess that hbase-shaded-server absorbed the class files.  You can verify it by looking at the pom as well:
https://github.com/apache/hbase/blob/rel/1.4.13/hbase-shaded/hbase-shaded-server/pom.xml#L53

hbase-shaded-server depends on hbase-server, which depends on hbase-client
hbase-shaded-server doesnt exclude it
hbase-shaded-server uses the shaded plugin w/o any exclusions

Reply:
Thanks @igorbernstein2 

I updated the pom file accordingly, PTAL.